kbfsmd/root_metadata_v3.go

```diff
@@ -208,12 +208,12 @@ func (extra ExtraMetadataV3) IsReaderKeyBundleNew() bool {
 // must be done separately.
 func MakeInitialRootMetadataV3(tlfID tlf.ID, h tlf.Handle) (
 	*RootMetadataV3, error) {
-	if tlfID.Type() != h.Type() {
+	if h.TypeForKeying() != tlf.TeamKeying && tlfID.Type() != h.Type() {
```

Comment:
We can also check that if `tlfID.Type() == tlf.SingleTeam` implies `h.Type() == tlf.SingleTeam`, when `h.TypeForKeying() == tlf.TeamKeying`, for completeness.